### PR TITLE
Added hint attribute to input field.

### PR DESCRIPTION
**Pull request by KerenChandran**
References #292 

Since no one was working on it, I thought I could give myself a challenge and add the hint text. I'm new to SCSS, so if there's a mistake then it would be great if you can tell me what I did wrong.

I also updated readme.md for input and added two new fields on the test page to show off the hint attribute in action (one with custom icon and another with multiline).

### CODE REVIEW

File: components/input/Input.jsx [hunk 1]
```diff
@@ -77,6 +79,7 @@ class Input extends React.Component {
         {icon ? <FontIcon className={style.icon} value={icon} /> : null}
         <span className={style.bar}></span>
         {labelText ? <label className={labelClassName}>{labelText}</label> : null}
+        {hint ? <label className={style.hint}>{hint}</label> : null}
```
Comment (javivelasco) [inline components/input/Input.jsx]: This is the only thing that sounds a little weird to me. Having a `label` tag element wrapping the hint text does not sound very semantic IMO. Shouldn't it be a span or something similar?

**Comment by KerenChandran** [comment]
You are right. I used the label as a starting point cause it had a similar behaviour to the hint attribute. I'll submit the change to a span in a few hours.

**Comment by javivelasco** [comment]
Alright!

**Comment by KerenChandran** [comment]
Hm... I don't understand as to why a simple change from label to span will cause npm install to fail. I even ran npm run test in my terminal and it passed all tests. Can you please help me look into this issue with Travis?

**Comment by KerenChandran** [comment]
Hm.. After looking at #376 and checking the docs, it appears the hint/helper text may need to appear below the input field. Should I modify the code to behave like that or leave it as is?

https://www.google.com/design/spec/components/text-fields.html#text-fields-required-fields

**Comment by renato** [comment]
The Text Field spec describes a "helper text" below the field one single time, in the required field section. All other sections in the same page puts a "hint text" inside the text field.

I think it's right to expect it to be inside the field.

**Comment by javivelasco** [comment]
Since it's an easy change, I think it causes to pain to keep it as it is right now. Sometimes MD looks confusing for me 😐

Thank you both!

**Comment by KerenChandran** [comment]
Fixes #292 

Sorry should have mentioned this before when submitting the ticket. Hopefully this closes the issue, otherwise @javivelasco can you close the issue?

**Comment by javivelasco** [comment]
Sure, thanks @KerenChandran

**Pull request merged**